.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -31,4 +31,8 @@ make version
 # or
 
 python -c "import autoimport.version; print(autoimport.version.version_info())"
+
+# or
+
+autoimport --version
```

Comment:
Thanks for the PR, I'd rather remove the L33 and substitute with yours, it's cleaner